style.css

```diff
@@ -137,7 +137,6 @@ nav {
 }
 
 .primary {
```

Comment:
- [ ] Kindly makes sure on mobile the `width` is set to `100%` as you can how your page looks on small screen
Your design :point_down: 

![image](https://user-images.githubusercontent.com/97100091/196740640-561501d4-fe38-43af-8b78-c79474acb5c2.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/196741007-8ca81995-add4-4c4c-9bbf-8bc011361eef.png)